docs: tw798 (#3270)
OffchainLabs/arbitrum-docs: docs/how-arbitrum-works/01-inside-arbitrum-nitro.mdx

  Transaction lifecycle
</ImageZoom>

> Along the way, you will learn about Arbitrum’s ability to deliver security on par with Ethereum, while achieving fee reductions by a factor of ten and transaction speeds accelerated by a factor of 100 using optimized mechanisms.

Reviewer: ```suggestion
Along the way, you'll learn about Arbitrum's ability to deliver security on par with Ethereum, while achieving fee reductions by a factor of ten and transaction speeds accelerated by a factor of 100 using optimized mechanisms.
```
